docs/man/docker-set.1.md

```diff
@@ -0,0 +1,62 @@
+% DOCKER(1) Docker User Manuals
+% Docker Community
+% JUNE 2014
+# NAME
+docker-set - Setup or update resource configs for one or more containers
+
+# SYNOPSIS
+**docker set**
+[**-c**|**--cpu-shares**[=*0*]]
+[**--cpuset-cpus**[=*CPUSET-CPUS*]]
+[**--help**]
+[**-m**|**--memory**[=*MEMORY*]]
+[**--memory-swap**[=*MEMORY-SWAP*]]
+CONTAINER [CONTAINER...]
+
+# DESCRIPTION
+
+Set resource configs for one or more containers. This can be the first
```

Comment:
Yes, it's not clear, and a bit redundant, I just get rid of it, do you have better suggestion?
Updated and rebased, thanks.